config/openhpc.yml

```diff
@@ -167,4 +167,5 @@ openhpc_packages:
   - git-lfs
   - perf
   - perl-devel
+  - gdb
```

Comment:
nit: If we ever audit all of these packages, it would be helpful if there was a comment in the commit message, or in the playbook explaining why it is there. I guess someone wants to debug something?

Reply:
I've mentioned who requested it in the PR message.

Reply:
My only issue with that is that when you look at the source it isn't easy to find the PR that the change came from, especially if it isn't referenced anywhere in the commit message.  If the reason for adding the package is in the commit message, then you can do a git blame and get some context really easily.